addons/test_mail/tests/test_mail_message.py

```diff
@@ -27,6 +27,62 @@ def setUpClass(cls):
         })
         cls.Message = cls.env['mail.message'].with_user(cls.user_employee)
 
+    def test_mail_message_empty(self):
+        """
+            Test that message is correctly considered as empty (`_filter_empty_messages()`).
+            Message considered as empty if there is:
+                - no body or empty body
+                - AND no subtype or no subtype description
+                - AND no tracking values
+                - AND no attachment
+        """
+        record1 = self.env['mail.test.simple'].create({'name': 'EmptyTesting'})
```

Comment:
Shouldn't we add some followers and test the effects of emptying the record (aka removing the notifications and starred users) ?